src/doc/trpl/README.md

```diff
@@ -145,20 +155,27 @@ fn main() {
 ^
 ```
 
-Whew! The Rust compiler gives quite detailed errors at times, and this is one
-of those times. As the error explains, while we made our binding mutable, we
-still cannot call `push`. This is because we already have a reference to an
-element of the vector, `y`. Mutating something while another reference exists
-is dangerous, because we may invalidate the reference. In this specific case,
-when we create the vector, we may have only allocated space for three elements.
-Adding a fourth would mean allocating a new chunk of memory for all those elements,
-copying the old values over, and updating the internal pointer to that memory.
-That all works just fine. The problem is that `y` wouldn’t get updated, and so
-we’d have a ‘dangling pointer’. That’s bad. Any use of `y` would be an error in
-this case, and so the compiler has caught this for us.
-
-So how do we solve this problem? There are two approaches we can take. The first
-is making a copy rather than using a reference:
+うひゃあ！
+Rust コンパイラはときどき極めて詳細なエラーを吐きます。これはそんなときのうちの一つです。
+エラーが説明するように、
+束縛をミュータブルにしたとしてもまだ `push` を呼ぶことはできません。
+これは、ベクトルの要素への参照 `y` を既に持っているからです。
```

Comment:
> This is because we already have a reference to an element of the vector, `y`
> これは、ベクトルの要素への参照 `y` を既に持っているからです。

「既に持っている」とするとちょっと勘違いしやすいので、「既にある」などのほうがいいかもしれません。